### DISCUSSION TITLE
Some question by a new user

### DISCUSSION THREAD

**Discussion by ejemba** [kovidgoyal/kitty#3172] [Q&A] [answered]
Hi o/ 

1.    When doing « kitty_mod+e kitten hints » how can I replace the action "open url in navigator" by "copy url in clipboard" ?
1.    Nothing happens when I click on a normal link → how can I debug this ?
1.    Nothing happens when I click on an image link (should display the image in overlay) → how can I debug this ?

thank you

**Comment by aorith** [comment, marked as answer]
Hey!

1. `map kitty_mod+e kitten hints --program @`  see https://sw.kovidgoyal.net/kitty/kittens/hints.html
2. Are you holding `kitty_mod`? also you can define another key with `open_url_modifiers <key>`  see https://sw.kovidgoyal.net/kitty/conf.html - Mouse section
3. I think you mean this: https://sw.kovidgoyal.net/kitty/open_actions.html

Regards,

**Comment by ejemba** [comment]
@aorith awesome !

Thank you very much
